### DISCUSSION TITLE
Some ideas from Georgia-ReBORN's Crazy Fans 风里尘

### DISCUSSION THREAD

**Discussion by Meisfuture** [TT-ReBORN/Georgia-ReBORN#181] [Ideas]
![123](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/74baa45c-9f0c-4c5d-83a0-fe87b35a71d3)

![A](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/af5859b0-2750-46a1-bec4-afb723a2dc04)

![B](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/d85b2a56-5d90-401d-8205-a5a4c645d611)

![AA5](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/08f1aa18-5fcf-42cc-8ea9-e290a2de5c80)

![CD](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/d7d2a5ee-0421-48c0-963a-209821ffb402)

![D](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/ec999ee2-4a0c-44b4-bf39-3a1ee54d3153)

![AA](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/f3cda8eb-d9de-4e9a-9532-30af00f6eb68)

**Comment by TT-ReBORN** [comment]
@Meisfuture,

thanks for posting! I really like the vinyl tonearm, I think I will implement this as an option soon.
Also you have some humor with your second screenshot, not sure if it's serious or a troll post :sweat_smile:

-Tom

**Reply by Meisfuture** [reply]
Tom, 风里尘 is a crazy guy.😂

That's him.
https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/da2cd130-9ba6-4354-bb0e-3313d390e1fb


![1](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/db36f143-0d91-460d-8a2e-a881a9ac8a92)

![2](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/bc50341a-60b0-499b-823a-c4d887db762a)

![Image_1715947023088.png](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/17525c46-6b3b-4c00-9f7b-1c94404b2422)

![Image_1715947026639.png](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/eb4091f0-6d95-4dd8-8f6b-425adc61aab0)

**Reply by TT-ReBORN** [reply]
It's always great to see new modifications to Georgia-ReBORN =)
